src/components/input/FilterDropdown.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+    <v-menu location="bottom" v-model="menuOpen" :close-on-content-click="false">
+        <template v-slot:activator="{ props: templateProps }">
+            <v-chip v-bind="templateProps" :color="chipColor" :variant="chipVariant">
+                <template #append>
+                    <v-icon
+                        v-if="model.length"
+                        class="ml-1"
+                        @click.stop="model = []"
+                        aria-label="remove"
+                        icon="mdi-close-circle"
+                    />
+                    <v-icon v-else class="ml-1" icon="mdi-menu-down" aria-hidden="true" />
+                </template>
+                {{ chipLabel }}
+            </v-chip>
+        </template>
+        <v-list max-height="300px" min-width="300px">
+            <div class="pa-2">
+                <v-text-field
+                    v-model="searchText"
+                    label="Search items..."
+                    density="compact"
+                    hide-details
+                    clearable
+                    prepend-inner-icon="mdi-magnify"
+                    variant="outlined"
+                    autofocus
+                ></v-text-field>
+            </div>
+            <v-divider></v-divider>
+            <template v-if="filteredItems.length > 0">
+                <v-list-item v-for="item in filteredItems" :key="item.id">
+                    <v-checkbox v-model="model" :value="item.id" density="compact" hide-details>
+                        <template #label>
+                            <slot :item="item">
+                                {{ item.name }}
+                            </slot>
+                        </template>
+                    </v-checkbox>
+                    <v-tooltip v-if="item.description" :text="item.description" activator="parent" />
+                </v-list-item>
+            </template>
+            <div v-else-if="filteredItems.length === 0" class="pa-4 text-center text-grey">No items found.</div>
```

Comment:
```suggestion
            <div v-else-if="filteredItems.length === 0" class="pa-4 text-center text-medium-emphasis">No items found.</div>
```

grey  is not registered, this is how grey text is done everywhere else in this project